### PR TITLE
Add methods to get the next version based on the current version

### PR DESCRIPTION
**Pull request by johscheuer**
# Description

This change adds methods to the `FDBVersion` struct to get the next versions. This will be useful e.g. for testing purposes.

## Type of change

*Please select one of the options below.*

- New feature (non-breaking change which adds functionality)

# Discussion

No

# Testing

Unit tests

# Documentation

No

# Follow-up

Nothing

### CODE REVIEW

File: api/v1beta1/foundationdb_version.go [hunk 1]
```diff
@@ -156,6 +156,21 @@ func (version FdbVersion) HasNonBlockingExcludes() bool {
...
+func (version FdbVersion) NextMajorVersion() FdbVersion {
+	return FdbVersion{Major: version.Major + 1, Minor: 0, Patch: 0}
+}
+
+// NextMinorVersion returns the next minor version of FoundationDB.
+func (version FdbVersion) NextMinorVersion() FdbVersion {
+	return FdbVersion{Major: version.Major, Minor: version.Minor + 1, Patch: 0}
+}
+
+// NextPatchVersion returns the next path version of FoundationDB.
```
Comment (ltsampros) [inline api/v1beta1/foundationdb_version.go:169]: typo

File: api/v1beta1/foundationdbbackup_types_test.go [hunk 2]
```diff
@@ -21,8 +21,6 @@
 package v1beta1
 
 import (
-	"github.com/onsi/gomega"
```
Comment (ltsampros) [inline api/v1beta1/foundationdbbackup_types_test.go:24]: Can we separate these changes to a separate PR?

**Review by ltsampros — CHANGES_REQUESTED** [review]
LGTM apart from two minor comments.

File: api/v1beta1/foundationdbbackup_types_test.go (hunk 2 above)
Comment (johscheuer) [reply api/v1beta1/foundationdbbackup_types_test.go:24]: Sure

**Review by ltsampros — APPROVED** [review]
LGTM.
